examples/flutter_gallery/test_driver/transitions_perf_test.dart

```diff
@@ -93,6 +93,7 @@ Future<void> saveDurationsHistogram(List<Map<String, dynamic>> events, String ou
 
   if (unexpectedValueCounts.isNotEmpty) {
     final StringBuffer error = StringBuffer('Some routes recorded wrong number of values (expected 2 values/route):\n\n');
+    error.write('You must add the --trace-startup parameter to run the test. \n\n');
```

Comment:
Can you please add a comment that explains why `--trace-startup` is needed just in case that someone is curious about why this error is printed out? A link to corresponding comments on the line 19 would be sufficient.

Reply:
> Can you please add a comment that explains why `--trace-startup` is needed just in case that someone is curious about why this error is printed out? A link to corresponding comments on the line 19 would be sufficient.

Thanks.
Can I just add this comment. The comments are as follows:
"When run with --trace-startup, the VM stores trace events in an endless buffer instead of a ring buffer."
Or，just link to line 19,  But I don't known how to add comment to link a line.

Reply:
"When run with --trace-startup, the VM stores trace events in an endless buffer instead of a ring buffer." is great!

Reply:
> "When run with --trace-startup, the VM stores trace events in an endless buffer instead of a ring buffer." is great!

Thanks.
Done.